odoo/addons/base/models/ir_asset.py

```diff
@@ -116,9 +119,9 @@ def _parse_bundle_name(self, bundle_name, debug_assets):
             if bundle_name.endswith('.rtl'):
                 bundle_name = bundle_name[:-4]
                 rtl = True
-        elif asset_type != 'js':
-            raise ValueError('Only js and css assets bundle are supported for now')
-        if len(bundle_name.split('.')) != 2:
+        elif asset_type not in ['js', 'binary']:
```

Comment:
```suggestion
        elif asset_type not in ('js', 'binary'):
```
one of our own guidelines in base is to use tuples instead of lists for this kind of inclusion-test